addons/edi/models/edi_document.py

```diff
@@ -249,10 +271,31 @@ def _compute_input_count(self):
 
     @api.depends("output_ids", "output_ids.res_id")
     def _compute_output_count(self):
-        """Compute number of output attachments (for UI display)"""
+        """Compute number of output attachments (for UI display)."""
         for doc in self:
             doc.output_count = len(doc.output_ids)
 
+    def _compute_prepared_count(self):
```

Comment:
nit: maybe _compute_prepared_count and _compute_progress_log can be combined as they share same (no) dependency